Model/Dintero.php

```diff
@@ -335,7 +341,7 @@ public function processOrder($order)
         $this->addStatusComment($payment);
         $order->save();
 
-        if (!$isFailed) {
+        if (!$isFailed && !$order->getEmailSent()) {
```

Comment:
Might be outdated If it happens in the same second?

Reply:
@andersem , I've added additional check for this case.